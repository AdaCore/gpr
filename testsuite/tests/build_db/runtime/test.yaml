description:
    Test runtime units resolution and checks
driver: build_and_run
project_file: test
main: main
