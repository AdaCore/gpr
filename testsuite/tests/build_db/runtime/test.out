Ada.Streams located
 dep: ADA
 dep: ADA.IO_EXCEPTIONS
